src/main/steps/upload-documents/content.test.ts

```diff
@@ -136,3 +139,16 @@ describe('generateContent() function Test', () => {
     expect(generateContent(content).form).toBe(form);
   });
 });
+
+describe('getErrors() function Test', () => {
+  test('getErrors English', () => {
+    const englishErrors = getErrors('en');
+    expect(englishErrors).toEqual(en().errors);
+  });
+
+  test('getErrors Welsh', () => {
+    const welshErrors = getErrors('cy');
+    console.log("3")
```

Comment:
Is there a reason for this `console.log` or can it be removed?

Reply:
No, that should have been removed. Thanks!